config/validator/service/proposer_settings.go

```diff
@@ -136,6 +147,35 @@ type ProposerOption struct {
 	BuilderConfig      *BuilderConfig
 }
 
+// Clone creates a deep copy of proposer option
+func (po *ProposerOption) Clone() *ProposerOption {
+	if po == nil {
```

Comment:
Is it even possible to enter into this function if `po == nil`?

Reply:
probably not i think... don't remember by we added those checks

Reply:
nvm it's used on fee recipient APIs reverted this...